### PR TITLE
Update pt_PT app translation for 3.8.2

### LINKED ISSUE #2308: Update pt_PT app translation for 3.8.2
Hi @Snayler,
We are getting ready for the 3.8.2 release. No further changes to translatable strings are currently planned for this release.

We would be happy if you updated the Jamulus software translations for **pt_PT** by **2022-02-06**.

Please
- Update your fork from `jamulussoftware/jamulus` `master` and create a working branch
- Update translations using Qt Linguist in your fork,
- Commit and push your changes and reference this Issue,
- Open a Pull Request before 2022-02-06

Suggested commit and PR message:
```
Update pt_PT app translation for 3.8.2

Fixes #2308
```

Note: Some of the displayed "untranslated" strings will actually not be new, but will rather be shown due to technical reformatting (e.g. %1). You may be able to easily update them using the "vanished" translations list.

Further documentation can be found in [TRANSLATING.md](https://github.com/jamulussoftware/jamulus/blob/master/TRANSLATING.md).

Thanks for contributing to Jamulus!

### PR DESCRIPTION
**Pull request by melcon**
Fixes #2308

### CODE REVIEW

File: src/res/translation/translation_pt_PT.ts [hunk 1]
```diff
@@ -1451,37 +1451,37 @@ O seu canal foi silenciado e foi activada a função &apos;Silenciar-me&apos;. P
...
         <location filename="../../clientsettingsdlg.cpp" line="282"/>
         <source>Language combo box</source>
-        <translation type="unfinished"></translation>
+        <translation>Seletor de idioma</translation>
     </message>
     <message>
         <location filename="../../clientsettingsdlg.cpp" line="291"/>
         <source>and</source>
-        <translation type="unfinished"></translation>
+        <translation></translation>
```
Comment (ann0see) [inline src/res/translation/translation_pt_PT.ts:1474]: Are you sure that approving an empty string is correct?

File: src/res/translation/translation_pt_PT.ts [hunk 2]
```diff
@@ -4247,33 +4247,33 @@ O seu canal foi silenciado e foi activada a função &apos;Silenciar-me&apos;. P
...
         <location filename="../../../windows/sound.cpp" line="76"/>
         <source>Couldn&apos;t initialise the audio driver. Check if your audio hardware is plugged in and verify your driver settings.</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Não foi possível inicializar o driver de áudio. Confirme se seu equipamento de áudio está conectado e verifique suas configurações do driver.</translation>
     </message>
     <message>
         <location filename="../../../windows/sound.cpp" line="146"/>
         <source>The selected audio device is incompatible since it doesn&apos;t support a sample rate of %1 Hz. Please select another device.</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">O dispositivo de áudio selecionado é incompatível uma vez que não suporta  uma taxa de amostras de %1. Por favor, selecione outro dispositivo.</translation>
```
Comment (ann0see) [inline src/res/translation/translation_pt_PT.ts:4260]:
```suggestion
        <translation type="unfinished">O dispositivo de áudio selecionado é incompatível uma vez que não suporta uma taxa de amostras de %1. Por favor, selecione outro dispositivo.</translation>
```

File: src/res/translation/translation_pt_PT.ts [hunk 3]
```diff
@@ -4314,82 +4314,82 @@ O seu canal foi silenciado e foi activada a função &apos;Silenciar-me&apos;. P
...
         <location filename="../../../linux/sound.cpp" line="210"/>
         <source>JACK was shut down. %1 requires JACK to run. Please restart %1 to start JACK again. </source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">JACK foi encerrado. %1 requer JACK para ser executado. Reinicie %1 para iniciar JACK novamente. </translation>
     </message>
     <message>
         <location filename="../../../mac/sound.cpp" line="113"/>
         <source>No sound card is available in your system. CoreAudio input AudioHardwareGetProperty call failed.</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Nenhuma placa de som está disponível em  seu sistema. A entrada do CoreAudio falhou na chamada AudioHardwareGetProperty.</translation>
```
Comment (ann0see) [inline src/res/translation/translation_pt_PT.ts:4347]:
```suggestion
        <translation type="unfinished">Nenhuma placa de som está disponível em seu sistema. A entrada do CoreAudio falhou na chamada AudioHardwareGetProperty.</translation>
```

File: src/res/translation/translation_pt_PT.ts [hunk 4]
```diff
@@ -4314,82 +4314,82 @@ O seu canal foi silenciado e foi activada a função &apos;Silenciar-me&apos;. P
...
     </message>
     <message>
         <location filename="../../../mac/sound.cpp" line="122"/>
         <source>No sound card is available in the system. CoreAudio output AudioHardwareGetProperty call failed.</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Nenhuma placa de som está disponível em  seu sistema. A saída do CoreAudio falhou na chamada AudioHardwareGetProperty.</translation>
```
Comment (ann0see) [inline src/res/translation/translation_pt_PT.ts:4352]: As above: probably a space too much. Please also check below

File: src/res/translation/translation_pt_PT.ts [hunk 5]
```diff
@@ -4314,82 +4314,82 @@ O seu canal foi silenciado e foi activada a função &apos;Silenciar-me&apos;. P
...
     </message>
     <message>
         <location filename="../../../mac/sound.cpp" line="284"/>
         <source>The currently selected audio device is no longer present. Please check your audio device.</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">O dispositivo de áudio selecionado não está mais presente. Verifique seu  dispositivo de áudio.</translation>
```
Comment (ann0see) [inline src/res/translation/translation_pt_PT.ts:4357]: Space

File: src/res/translation/translation_pt_PT.ts [hunk 6]
```diff
@@ -4314,82 +4314,82 @@ O seu canal foi silenciado e foi activada a função &apos;Silenciar-me&apos;. P
...
         <location filename="../../../mac/sound.cpp" line="385"/>
         <source>The audio input device is no longer available. Please check if your input device is connected correctly.</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">O dispositivo de entrada de áudio não está mais disponível. Verifique se seu dispositivo de entrada está corretamente conectado.</translation>
     </message>
     <message>
         <location filename="../../../mac/sound.cpp" line="394"/>
         <source>The sample rate on the current input device isn&apos;t %1 Hz and is therefore incompatible. Please select another device or try setting the sample rate to %1 Hz manually via Audio-MIDI-Setup (in Applications-&gt;Utilities).</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">A taxa de amostragem no dispositivo de entrada atual não é %1 Hz e, portanto, é incompatível. Selecione outro dispositivo ou tente definir manualmente a taxa de amostragem para %1 Hz via Configuração de Áudio e MIDI(em Aplicativos-&gt;Utilitários).</translation>
```
Comment (ann0see) [inline src/res/translation/translation_pt_PT.ts:4367]:
```suggestion
        <translation type="unfinished">A taxa de amostragem no dispositivo de entrada atual não é %1 Hz e, portanto, é incompatível. Selecione outro dispositivo ou tente definir manualmente a taxa de amostragem para %1 Hz via Configuração de Áudio e MIDI (em Aplicativos-&gt;Utilitários).</translation>
```

File: src/res/translation/translation_pt_PT.ts [hunk 7]
```diff
@@ -4314,82 +4314,82 @@ O seu canal foi silenciado e foi activada a função &apos;Silenciar-me&apos;. P
...
         <location filename="../../../mac/sound.cpp" line="406"/>
         <source>The audio output device is no longer available. Please check if your output device is connected correctly.</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">O dispositivo de saída de áudio não está mais disponível. Verifique se seu dispositivo de saída está corretamente conectado.</translation>
     </message>
     <message>
         <location filename="../../../mac/sound.cpp" line="415"/>
         <source>The sample rate on the current output device isn&apos;t %1 Hz and is therefore incompatible. Please select another device or try setting the sample rate to %1 Hz manually via Audio-MIDI-Setup (in Applications-&gt;Utilities).</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">A taxa de amostragem no dispositivo de saída atual não é %1 Hz e, portanto, é incompatível. Selecione outro dispositivo ou tente definir manualmente a taxa de amostragem para %1 Hz via Configuração de Áudio e MIDI(em Aplicativos-&gt;Utilitários).</translation>
```
Comment (ann0see) [inline src/res/translation/translation_pt_PT.ts:4377]: As above. Insert a space?

File: src/res/translation/translation_pt_PT.ts [hunk 8]
```diff
@@ -4488,57 +4488,58 @@ O seu canal foi silenciado e foi activada a função &apos;Silenciar-me&apos;. P
...
         <location filename="../../util.cpp" line="1971"/>
         <source>Some pixmaps are from the Open Clip Art Library (OCAL)</source>
-        <translation type="unfinished"></translation>
+        <translation>Alguns pixmaps são da Open Clip Art Library (OCAL)</translation>
     </message>
     <message>
         <location filename="../../util.cpp" line="1974"/>
         <source>Flag icons by Mark James</source>
-        <translation type="unfinished"></translation>
+        <translation>Ícones de bandeiras por Mark James</translation>
```
Comment (ann0see) [inline src/res/translation/translation_pt_PT.ts:4542]: The first change lets this unfinished while this one is marked as finished.

**Review by ann0see — COMMENTED** [review]
Thank you very much. Please see my comments

File: src/res/translation/translation_pt_PT.ts (hunk 1 above)
Comment (melcon) [reply src/res/translation/translation_pt_PT.ts:1474]: Nope. I'll fix it. Not sure how it was lost

File: src/res/translation/translation_pt_PT.ts (hunk 8 above)
Comment (melcon) [reply src/res/translation/translation_pt_PT.ts:4542]: marked both as finished

File: src/res/translation/translation_pt_PT.ts (hunk 2 above)
Comment (melcon) [reply src/res/translation/translation_pt_PT.ts:4260]: done

File: src/res/translation/translation_pt_PT.ts (hunk 3 above)
Comment (melcon) [reply src/res/translation/translation_pt_PT.ts:4347]: done

File: src/res/translation/translation_pt_PT.ts (hunk 4 above)
Comment (melcon) [reply src/res/translation/translation_pt_PT.ts:4352]: done

File: src/res/translation/translation_pt_PT.ts (hunk 5 above)
Comment (melcon) [reply src/res/translation/translation_pt_PT.ts:4357]: done

File: src/res/translation/translation_pt_PT.ts (hunk 6 above)
Comment (melcon) [reply src/res/translation/translation_pt_PT.ts:4367]: done

**Review by hoffie — APPROVED** [review]
Thanks a lot for this additional work! :)

- [x] Translator listed in the src/util.cpp, optionally add link to PR or code
- [x] Punctuation and spacing consistent.
- [x] Signal words consistent (“ASIO”, “Buffer”)
- [ ] App translations: No untranslated strings (grep unfinished -5 src/res/translation/translation_$TRANSLATION*.ts)* (deliberately left as unfinished for cross-checking by main translator for pt_PT)
- [x] App translations: Only a single .ts file checked in (.qm in addition is also OK)
- [x] Installer translations

@ann0see Should be squashed.

**Review by ann0see — APPROVED** [review]

**Pull request merged**